bin/node/cli/src/command.rs

```diff
@@ -79,6 +79,7 @@ pub fn run() -> Result<()> {
 
 			runner.sync_run(|config| cmd.run::<Block, RuntimeApi, Executor>(config))
 		}
+		#[cfg(feature = "runtime-benchmarks")]
```

Comment:
Yeah, I think it is a good idea to improve the message. We should remove the cfg here and in the enum declaration.

If the feature is given, it should call the code below and if the feature is not given, it should print that benchmarking wasn't enabled when building the node.